tools/src/wyvern/stdlib/support/Stdio.java

```diff
@@ -1,5 +1,6 @@
 package wyvern.stdlib.support;
 
+import java.math.BigDecimal;
```

Comment:
Is there a strong motivation for using BigDecimal?  I used BigInteger for Int because of all the problems that overflow causes in languages with fixed-width integers.  But I don't see the same obvious problems with floating point.  What about making a simple built-in double the standard representation in Wyvern?